### PR TITLE
Ubuntu 2204

### PR DESCRIPTION
**Pull request by aravindavk**

### CODE REVIEW

File: .github/workflows/on-pr-merge-ubuntu-2204.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+jobs:
+  devel-tag-push:
+    runs-on: ubuntu-22.04
+    steps:
+    - uses: actions/checkout@v2
+    # TODO: Review the below list of dependencies
+    - name: Install the Build dependencies
+      run: |
+        sudo apt update -y
```
Comment (erik-bershel) [inline .github/workflows/on-pr-merge-ubuntu-2204.yaml:20]: If you add the line: "sudo apt install -y libunwind-dev"
between "sudo apt update -y" and "sudo apt install -y --no-install-recommends...." all would work.
Example and test run are [here](https://github.com/erik-bershel/test-repo-pipelines/actions/workflows/example.yml).

Comment (aravindavk) [reply .github/workflows/on-pr-merge-ubuntu-2204.yaml:20]: Thanks for the help.

Comment (pieter3d) [reply .github/workflows/on-pr-merge-ubuntu-2204.yaml:20]: I ran into this same issue, and this solved it for me as well (the libunwind-14 and libunwind-dev conflict). But I don't really understand why? Is that some issue with something that is depending on libunwind? And why would updating them in two different steps solve the problem? What prevents this from happening again? It's kind of awkward to have this explicit update in my CI without any real explanation.

Comment (erik-bershel) [reply .github/workflows/on-pr-merge-ubuntu-2204.yaml:20]: Hi @pieter3d!
[Here](https://bugs.launchpad.net/ubuntu/+source/google-glog/+bug/1991919) is the explanation. It is just a related to dependencies bug. Nobody can prevent it.

**Pull request merged**